sfcasts/intro.md

```diff
@@ -1,70 +1,43 @@
 # Introduction
 
-Welcome back friends to episode 2 of our design patterns series! In this episode,
-we'll continue our journey of building the *greatest* command-line RPG game ever!
-And, for this challenging task we'll apply 5 new design patterns.
+Hey friends! Welcome back to episode *two* of our Design Patterns series! In this episode, we'll continue our journey of building the *greatest* command-line RPG game *ever*! To do that, we'll apply not *one*, not *two*, but *five* new design patterns.
 
-We'll learn three new *behavioral* patterns, *Command*, *Chain of Responsibility*,
-and *State*. These patterns help organize code into separate classes that can
-then interact with each other.
+We'll learn three new *behavioral* patterns: *Command*, *Chain of Responsibility*, and *State*. These patterns help organize code into separate classes that can then interact with each other.
 
-Also, we'll learn about the *Factory* pattern, which is a *creational* pattern.
-These type of patterns are all about helping instantiate objects, as we've already
-seen with the *builder pattern* in episode one. And, as a bonus we'll cover
-one of my favorites, the *NullObject* pattern.
+We'll also learn about the *Factory* pattern, which is a *creational* pattern. This type of pattern is all about helping instantiate objects, just like the *builder* pattern in episode one. And, as a bonus, we'll cover one of my favorites - the *NullObject* pattern.
 
-For more information about *types* of design patterns, I recommend you to watch
-the first chapter of the previous episode.
+For more information about design patterns, check out the first chapter of the episode one.
```

Comment:
the word `types` is required here because there actually are types of patterns, behavioral, structural, etc. So, what does sound better, "types of design patterns" or "design patterns' types"?

Reply:
Ah, okay. I think "design pattern types" would sound best here then. It also looks like my wording got a little wonky there. With this correction, it should read:

"For more information about design pattern types, check out the first chapter of episode one."